io.embrace.sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "io.embrace.sdk",
-  "version": "1.25.3",
+  "version": "1.27.0",
```

Comment:
This should probably be a major version bump given there are breaking changes to the Unity interface?

Reply:
I talked with Eliab about this, and we have more breaking changes incoming. I'm think we're open to being convinced otherwise though. Should we be looking at more of a 2.0 beta that doesn't have a stable API yet?

Reply:
I'd suggest discussing with Eliab again as you'll definitely have more context than me on what a good versioning strategy is